locale/fi/LC_MESSAGES/django.po

```diff
@@ -985,7 +985,7 @@ msgstr "Avaa 0 arvosanan diplomi"
 
```

Comment:
In general the appearance, code and the UX is good! I also did not have those issues anymore with the static files not loading, so that is solved. Just a few things there might be worth checking that I mentioned below, after those, I consider this done.

Reply:
> after those, I consider this done.

Asuming those got resolved, can you verify and mark the review "accepted"?